src/scripts/models/search-results.coffee

```diff
@@ -56,7 +56,8 @@ define (require) ->
 
     fetch: () ->
       fetchArgs = arguments[0]
-      fetchArgs.timeout = 60000 # one minute
+      if fetchArgs isnt undefined
+        fetchArgs.timeout = 60000 # one minute
```

Comment:
This could be written as `fetchArgs?.timeout = 60000`, or perhaps even better, set a default value to assign to the args:
`fetch: (fetchArgs = {timeout: 60000}) ->`